full-service/src/service/t3_sync.rs

```diff
@@ -0,0 +1,152 @@
+use mc_common::logger::{log, Logger};
+
+use t3_api::{T3Uri, TransparentTransaction};
+use t3_connection::T3Connection;
+
+use std::{
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        Arc,
+    },
+    thread,
+};
+
+use clap::Parser;
+
+use crate::{
+    db::{
+        models::{AuthenticatedSenderMemo, Txo},
+        txo::{TxoMemo, TxoModel},
+        Conn,
+    },
+    error::T3SyncError,
+    WalletDb,
+};
+
+#[derive(Clone, Debug, Parser)]
+pub struct T3Config {
+    #[clap(long, env = "T3_URI")]
+    pub t3_uri: T3Uri,
+    #[clap(long, env = "T3_API_KEY")]
+    pub t3_api_key: String,
+}
+
+// How many txos to sync per round
+const TXO_CHUNK_SIZE: usize = 5;
```

Comment:
any reason not to make this bigger?

Reply:
I left it small during testing so I could iterate without resetting my database. Not sure there's much of a reason to keep it small, do you have a preference? It is really only impactful during the initial sync, which doesn't take long as it is. Under normal operation, it should only be syncing a couple txos at most at any given time.

Reply:
5 every 10msec seems fine to me